BusinessProcessTrackingAlerts/manifest.json

```diff
@@ -0,0 +1,110 @@
+{
+    "title": "Business Process Tracking Alert Notifications",
+    "description": "The Business Process Tracking Alerts template allows you to detect Business Process Tracking stages that have failed so that you can notify key business stakeholders using Microsoft Teams.",
+    "prerequisites": "To use this template, you should have an existing Business Process deployed that is emitting data to an Azure Data Explorer (ADX) instance. This workflow will query a defined ADX instance and issue queries against that database and subsequently surface the business process details. To learn more about Busness Process Tracking, please see our documentation. [Learn more] (https://learn.microsoft.com/en-us/azure/business-process-tracking/create-business-process) You will also need an Azure Storage Account that includes a table, with the name of your choice, that includes a field called 'LastAlertedProcessed'",
+    "tags": [
+        "Automation"
+    ],
+    "skus": [
+        "standard"
+    ],
+    "kinds": [
+        "stateful",
+        "stateless"
+    ],
+    "detailsDescription": "This template is built upon AI building blocks in Logic Apps such as Azure OpenAI connector, Azure AI Search connector, and more. The template also uses built in actions to document parsing and chunking. [Learn more] (https://learn.microsoft.com/azure/logic-apps/connectors/azure-ai)",
+    "details": {
+        "By": "Microsoft",
+        "Type": "Workflow",
+        "Trigger": "Request",
+        "Category": "Automation"
+    },
+    "artifacts": [
+        {
+            "type": "workflow",
+            "file": "workflow.json"
+        },
+        {
+            "type": "maps",
+            "file": "BPKQL-to-Teams.xslt"
+        },
+        {
+            "type": "schemas",
+            "file": "BPKQL.json"
+        },
+        {
+            "type": "schemas",
+            "file": "team.json"
+        }
+    ],
+    "images": {
+        "light": "BusinessProcessTrackingAlertsWorkflow-light",
+        "dark": "BusinessProcessTrackingAlertsWorkflow-dark"
+    },
+    "parameters": [
+        {
+            "name": "Azure-TableName_#workflowname#",
```

Comment:
Everything that contains _#workflowname# needs to have those in the workflow.json as well